<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceBankManagerWindow</class>
 <widget class="QMainWindow" name="VoiceBankManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音源管理器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0">
    <item row="0" column="0">
     <widget class="QLabel" name="voicebankListTitleLabel">
      <property name="text">
       <string>音源库列表：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTableView" name="voiceBanksTableView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="voicebankCountLabel">
      <property name="text">
       <string>共 个</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="searchLineEdit">
      <property name="placeholderText">
       <string>键入以搜索</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_preference">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actionMonitor_Folders"/>
    <addaction name="actionDefault_TextCodec"/>
    <addaction name="actionSet_Thread_Pool_Max_Count"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionEdit_Global_MoresamplerConfig"/>
    <addaction name="actionchoose_a_voicebank_randomly"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_preference"/>
   <addaction name="menuTools"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="voiceBankBriefInfomationWidget">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="voicebankIconWidget" native="true">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="voiceIconLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="voicebankImage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playSamplebutton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>播放样例</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="voiceInformationWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="voicebankNameLabel">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>音源库名称</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="voicebankReadmeTextBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionImport_VoiceBank">
   <property name="text">
    <string>导入音源库...</string>
   </property>
   <property name="toolTip">
    <string>导入一个外部音源库用于管理</string>
   </property>
   <property name="statusTip">
    <string>导入一个外部音源库用于管理</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>重载</string>
   </property>
   <property name="toolTip">
    <string>重新载入监视文件夹下的音源库</string>
   </property>
   <property name="statusTip">
    <string>重新载入监视文件夹下的音源库</string>
   </property>
  </action>
  <action name="actionSet_chosen_voicebank_s_oto_ini">
   <property name="text">
    <string>对选定的音源库进行原音设定（oto.ini）...</string>
   </property>
   <property name="toolTip">
    <string>在原音设定编辑器中编辑该音源库的oto.ini</string>
   </property>
   <property name="statusTip">
    <string>在原音设定编辑器中编辑该音源库的oto.ini</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionMonitor_Folders">
   <property name="text">
    <string>监视文件夹...</string>
   </property>
   <property name="toolTip">
    <string>修改程序寻找音源的位置</string>
   </property>
   <property name="statusTip">
    <string>修改程序寻找音源的位置</string>
   </property>
  </action>
  <action name="actionDefault_TextCodec">
   <property name="text">
    <string>默认文本编码...</string>
   </property>
   <property name="toolTip">
    <string>修改程序读取音源文件时默认使用的编码</string>
   </property>
   <property name="statusTip">
    <string>修改程序读取音源文件时默认使用的编码</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于本程序</string>
   </property>
   <property name="statusTip">
    <string>关于本程序</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>关于 Qt</string>
   </property>
   <property name="toolTip">
    <string>关于本程序使用的 Qt 框架</string>
   </property>
   <property name="statusTip">
    <string>关于本程序使用的 Qt 框架</string>
   </property>
  </action>
  <action name="actionSet_Thread_Pool_Max_Count">
   <property name="text">
    <string>设置读取用线程池的最大大小...</string>
   </property>
   <property name="toolTip">
    <string>（高级）改变程序读取音源库时的最大线程数。请确保您在知道自己在做什么之后再更改此项设置。</string>
   </property>
   <property name="statusTip">
    <string>（高级）改变程序读取音源库时的最大线程数。请确保您在知道自己在做什么之后再更改此项设置。</string>
   </property>
  </action>
  <action name="actionEdit_Global_MoresamplerConfig">
   <property name="text">
    <string>编辑全局Moresampler设置</string>
   </property>
   <property name="toolTip">
    <string>在编辑器中打开一个moreconfig.txt并以全局模式编辑</string>
   </property>
   <property name="statusTip">
    <string>在编辑器中打开一个moreconfig.txt并以全局模式编辑</string>
   </property>
  </action>
  <action name="actionchoose_a_voicebank_randomly">
   <property name="text">
    <string>随机选择一个音源</string>
   </property>
   <property name="toolTip">
    <string>在音源列表中随机选择一个音源。在拿不定主意的时候说不定会有帮助。</string>
   </property>
   <property name="statusTip">
    <string>在音源列表中随机选择一个音源。在拿不定主意的时候说不定会有帮助。</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
